vendor/knative.dev/pkg/test/security/security.go

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
This will be reverted when running `make generate-release RELEASE=v1.5` because we re-pull dependencies to check consistency: https://github.com/openshift-knative/eventing-kafka-broker/blob/4ca1530eae3d0437d60b4bc84da4cf43dc8ccbe3/openshift/release/generate-release.sh#L7-L11, so we need a patch file for vendor dir changes.

Reply:
I see, a patch file...

Reply:
Pushed the patch and included it in generate-release.sh